### PR TITLE
Update constant queries and mutations names to CONSTANT_CASE

### PR DESCRIPTION
**Pull request by Jordanlelay**

### CODE REVIEW

**Review by bbenoist — APPROVED** [review]
Code looks goods to me apart from minor changes which I've already fixed when working on #834 (_a few mutations constants names with the `_QUERY` suffix)_.
Let's merge this PR into delete-service-worker right after we reintegrate #834 into this PR.

File: typescript/web/src/pages/[workspaceSlug]/datasets/index.tsx [hunk 1]
```diff
@@ -24,30 +24,7 @@ import { CookieBanner } from "../../../components/cookie-banner";
...
-      labelsAggregates {
-        totalCount
-      }
-      labelClassesAggregates {
-        totalCount
-      }
-    }
-  }
-`;
+import { GET_DATASETS_QUERY } from "../../../utils/shared-queries";
```
Comment (valerian) [inline typescript/web/src/pages/[workspaceSlug]/datasets/index.tsx:28]: This query is too specific to put it in shared queries with such a generic name: it's very specifically tailored for the workspace page listing datasets. with the thumbnail of the first image of a dataset.

I think it should be renamed at least, and then I'm not sure if it should be in shared queries or not.

File: typescript/web/src/utils/shared-queries/index.ts [hunk 2]
```diff
@@ -11,3 +11,27 @@ export const userQuery = gql`
     }
   }
 `;
+
+export const GET_DATASETS_QUERY = gql`
```
Comment (valerian) [inline typescript/web/src/utils/shared-queries/index.ts:15]: see previous comment

File: typescript/web/src/connectors/undo-store/effects/update-label.ts [hunk 3]
```diff
@@ -31,7 +26,17 @@ const updateLabelMutation = gql`
...
+    image(where: { id: $id }) {
+      id
+      width
+      height
+    }
+  }
+`;
+
+const GET_LABEL_QUERY = gql`
   query getLabelAndGeometry($id: ID!) {
```
Comment (valerian) [inline typescript/web/src/connectors/undo-store/effects/update-label.ts:40]: So you do that here, but @bbenoist undos it in #834, who is right here, and why?

**Review by valerian — CHANGES_REQUESTED** [review]

**Review by valerian — CHANGES_REQUESTED** [review]

**Pull request closed without merging**